src/Microsoft.AspNet.OData.Shared/Query/Expressions/NamedPropertyExpression.cs

```diff
@@ -34,6 +34,8 @@ public NamedPropertyExpression(Expression name, Expression value)
         // and use Value only if IsNull is false.
         public Expression NullCheck { get; set; }
 
+        public Expression RawValue { get; set; }
```

Comment:
>RawValue [](start = 26, length = 8)

what's the difference between "Value" and "RawValue"?

Reply:
Value might contain not the actual instance but rather wrapper created in SelectExpandBinder method BuildExpandedProperty (line 712).